pkgs/servers/http/apache-modules/mod_ca/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchurl, gnused, coreutils, pkgconfig, apacheHttpd, openssl, openldap }:
```

Comment:
`gnused` and `coreutils` won't be required, see below.

Reply:
you are quite right - minimized now.